Lib/pickletools.py

```diff
@@ -1894,8 +1915,11 @@ def __init__(self, name, code, arg,
       code registry ought to be global, although a range of codes may
       be reserved for private use.
 
-      EXT1 has a 1-byte integer argument.  This is used to index into the
-      extension registry, and the object at that index is pushed on the stack.
+      EXT1 has a 1-byte integer argument.  This is used to index into
+      the inverted extension registry, which contains integer to tuple
+      mappings. The tuples have a length of two in the format of
+      '("module", "name")'. This tuple is then passed through find_class,
+      and the result is pushed onto the stack.
```

Comment:
It mentions too much implementation details (the inverted extension registry and its format). It should only refer to `copyreg.add_extension()` (which unfortunately is not documented) and `find_class()` (like in `GLOBAL`, or just refer to it).